authentication/authentication.go

```diff
@@ -102,6 +102,7 @@ func (ah *ProviderManager) InitializeProvider(config map[string]interface{},
 	return authCh
 }
 
+// Middleware returns an authentication middleware for a tenant.
```

Comment:
Thanks for fixing these @matej-g. Middleware should be Middlewares

Reply:
Ah missed that, thanks! Will fix it.

Reply:
ping here